maven-release-manager/src/main/java/org/apache/maven/shared/release/exec/ForkedMavenExecutor.java

```diff
@@ -110,6 +110,9 @@ public void executeGoals( File workingDirectory, List<String> goals, ReleaseEnvi
 
             cl.setWorkingDirectory( workingDirectory.getAbsolutePath() );
 
+            // FIX for MRELEASE-1105
+            //cl.addEnvironment( "MAVEN_DEBUG_OPTS", "" );
+
```

Comment:
probably for next PR - unneeded comments

Reply:
That's why it's here as comment. To remember where change that. As I will probably work on next PR in 1 or 2 week so I can definitely forget 🤣 or someone else pick the other issue..
I do not see this as a problem it's a comment and probably helpful one 😉